translations/messages.fr.xlf

```diff
@@ -0,0 +1,3059 @@
+        <source>Sign up</source>
+        <target>S&apos;inscrire</target>
+      </trans-unit>
+      <trans-unit id="2715185765061607307" datatype="html">
+        <source> Log in </source>
+        <target> Se connecter </target>
+      </trans-unit>
+      <trans-unit id="4768749765465246664" datatype="html">
+        <source>Email</source>
+        <target>Email</target>
+      </trans-unit>
+      <trans-unit id="1431416938026210429" datatype="html">
+        <source>Password</source>
+        <target>Mot de passe</target>
+      </trans-unit>
+      <trans-unit id="5939693457139759683" datatype="html">
+        <source>A confirmation email will be sent to validate your account</source>
+        <target>Un email de confirmation vous sera envoyé pour valider votre compte</target>
+      </trans-unit>
+      <trans-unit id="77522255637065336" datatype="html">
+        <source>Connection error</source>
+        <target>Erreur de connexion</target>
+      </trans-unit>
+      <trans-unit id="2336550011721758066" datatype="html">
+        <source>Log in</source>
+        <target>Connexion</target>
+      </trans-unit>
+      <trans-unit id="3119050750831891643" datatype="html">
+        <source>This password is incorrect.</source>
+        <target>Ce mot de passe est incorrect</target>
+      </trans-unit>
+      <trans-unit id="5113647226865218037" datatype="html">
+        <source>This email address has no account on this website.</source>
+        <target>Cette adresse email n&apos;a pas de compte sur ce site.</target>
+      </trans-unit>
+      <trans-unit id="1988081660757251720" datatype="html">
+        <source>You must click the confirmation link that you should have received by email.</source>
+        <target>Vous devez aller dans vos mail confirmer votre inscription en cliquant sur le lien que nous vous avons envoyé d&apos;abord.</target>
+      </trans-unit>
+      <trans-unit id="850080272338290812" datatype="html">
+        <source>No account?</source>
+        <target>Pas de compte ?</target>
+      </trans-unit>
+      <trans-unit id="8999284073934392732" datatype="html">
+        <source>Login with Google</source>
+        <target>Se connecter avec Google</target>
+      </trans-unit>
+      <trans-unit id="7762039054063004692" datatype="html">
+        <source> The replay game is loading, please wait, it should not take long.</source>
+        <target> Le partie de revanche se charge. Veuillez attendre, cela ne devrait pas prendre longtemps.</target>
+      </trans-unit>
+      <trans-unit id="5120671221766405888" datatype="html">
+        <source>Game does not exist</source>
+        <target>Partie inexistante</target>
+      </trans-unit>
+      <trans-unit id="5769704000858519890" datatype="html">
+        <source>The game you tried to join does not exist anymore.</source>
+        <target>La partie que vous avez essayé de rejoindre n&apos;existe pas ou n&apos;existe plus !</target>
+      </trans-unit>
```

Comment:
Not the same meaning.

Reply:
Changé en "n'existe plus", mais améliorable je dirais ! Je vois trois scénarios dans lequel ce message peut s'afficher, tous étant couverts par la version FR. Cependant la version EN correspondante ferait "does not exist or does not exist anymore" c'qu'est un peu moche.

  - Tu m'envoies un lien que je clique trop tard, la partie a déjà été finie -> *La partie n'existe plus*
  - Tu m'envoies un lien mais tu le bork, donc je me retrouve avec un lien incorrect -> *La partie n'existe pas*
  - J'essaye de deviner un lien aléatoire -> *La partie n'existe pas*